client/src/views/Feed/Feed.vue

```diff
@@ -94,7 +99,12 @@ export default {
     try {
       if (this.$route.query.tab === 'search') {
         this.isSearch = true
-        this.getPostsAction({ type: 'search', tags: this.$route.query.tags })
+        if (this.$route.query.tags) {
+          this.getPostsAction({ type: 'search', tags: this.$route.query.tags })
+        }
+        if (this.$route.query.emoji) {
```

Comment:
Technically there is a case when both emoji and tags are available - in current case you will receive 2 subsequent search action with one replacing another (if i got this right). To prevent this there should have been a return statement after single action is invoked.